nix/cbmc/default.nix

```diff
@@ -1,36 +1,22 @@
 # SPDX-License-Identifier: Apache-2.0
 { buildEnv
 , cbmc
-, fetchFromGitHub
 , callPackage
 , bitwuzla
 , ninja
 , cadical
-, libgcc
 , z3
 }:
 
 buildEnv {
   name = "pqcp-cbmc";
   paths =
     builtins.attrValues {
-      cbmc = cbmc.overrideAttrs (old: rec {
```

Comment:
We are updating cbmc quite often - can you please keep the fixed version so that it is easy to update the version number?